projects/web-wallet/rpc-protocol.md

```diff
@@ -0,0 +1,330 @@
+# 1. Introduction
+
+This document outlines the remote procedure call (RPC) methods for a decentralized application (dApp) to interact with Hathor wallets.
+
+The design is currently a minimum viable product (MVP), addressing only known needs from our community. It should be noted that this API design may change as the platform evolves and additional needs arise.
+
+The idea here is to partially replicate the headless wallet API as it is already being used by many partners and has been validated to work on their use cases
+
+# 2. Reference-level explanation
```

Comment:
I feel this should be a thorough document. So we should add all RPC methods, including the ones already available in the POC.

Reply:
We only have the `signMessage` method in the POC

Reply:
Got it. Thanks!